.github/workflows/build_wheels_windows.yml

```
@@ -0,0 +1,157 @@
+        run: |
+          choco install -y cmake llvm uv rustup.install
+          # Install vcpkg for FFmpeg development libraries
+          git clone https://github.com/microsoft/vcpkg.git C:\vcpkg
+          cd C:\vcpkg
+          .\bootstrap-vcpkg.bat
+          .\vcpkg.exe install ffmpeg:x64-windows
+          # Set VCPKG_ROOT for cargo to find FFmpeg libraries
+          echo "VCPKG_ROOT=C:\vcpkg" >> $env:GITHUB_ENV
+
+      - name: Build Python wheels
+        run: |
```

actionlint (1.7.11)

[error] shellcheck reported issue in this script: SC1001:info:4:27: This \o will be a regular 'o' in this context

(shellcheck)